.eslintrc

```diff
@@ -8,7 +8,8 @@
   },
 
   "rules": {
-    "react-native/no-color-literals": "off"
+    "react-native/no-color-literals": "off",
+    "react/no-did-update-set-state": "off"
```

Comment:
See comment below for how to avoid this. And just a general tip: it's preferable to type `// eslint-disable-next-line react/no-did-update-set-state` above your code instead of setting a global rule 👍